### ISSUE TITLE
Populate the demos array on element in generateAnalysis from jsdoc

### ISSUE THREAD

**Issue by rictic** [Polymer/polymer-analyzer#578]
e.g.

```javascript
/**
 * @demo demo.html
 */
```

should become `demo: ["demo.html"]`

**Comment by samuelli** [comment]
FWIW, there already seems to be a `demos: []` on the element from `generateAnalysis()` but it always seems to be empty.

**Comment by usergenic** [comment]
Should the `@demo` annotation be picked up outside a JSDoc like in HTML comment such as in this case in `polymer-analyzer/src/test/static/analysis/simple/simple-element.html`

```
<!--
This is a description of the element.
@demo demo.html
-->
<script>
  Polymer({
    properties: {
      /**
       * This is a simple test property named theValue.
       */
      theValue: {
        type: String,
        notify: true
      },
      _protectedValue: String,
...
```

**Comment by rictic** [comment]
Yes, I think we should pick it up in either an html comment or a js comment. I believe currently in the code the `jsdoc` property on an element will come from either html or js, but not both. In theory this should be as simple as finding the right spot and extracting the demos from element.jsdoc

**Comment by rictic** [comment]
That spot might be in generateAnalysis, though it might also be useful to have demos on Element, Mixin, and Behavior

**Comment by samuelli** [comment]
@usergenic @rictic: any update on this? Need this for parity on webcomponents.org

**Comment by usergenic** [comment]
Working on this one today.  Hopefully will have it soon.

**Comment by justinfagnani** [comment]
Looks fixed by #590
